iis/datadog_checks/iis/iis.py

```diff
@@ -48,6 +48,15 @@ class IIS(PDHBaseCheck):
     def __init__(self, name, init_config, agentConfig, instances):
         PDHBaseCheck.__init__(self, name, init_config, agentConfig, instances=instances, counter_list=DEFAULT_COUNTERS)
 
+    def get_iishost(self, instance):
+        inst_host = instance.get("host")
+        if inst_host in [".", "localhost"]:
+            # Use agen'ts hostname if local
+            iis_host = self.hostname
+        else:
+            iis_host = inst_host
+        return "iis_host:{}".format(self.normalize(iis_host))
```

Comment:
Can `inst_host ` be `None`?

Reply:
seems that it can and behaves like `host: "."`. Will make some adjustments.